### ISSUE TITLE
deck control capture fail

### ISSUE THREAD

**Issue by bturkus** [amiaopensource/vrecord#572]
I've been trying deck control, thinking about how it would get incorporated into general capture practice, and I think there's a bug when attempting capture after getting a tape going with RS-422. 

So deck control is turned on, I've got capture dir set, my recording named, etc., and when I hit play with deck control and quickly push the record button, I get this error:

```
(base) MY-LPAMI-056430:~ benjaminturkus$ vrecord -e
/usr/local/bin/vrecord: line 1178: Issued: command not found
/usr/local/bin/vrecord: line 1179: Issued: command not found
/usr/local/bin/vrecord: line 1180: Issued: command not found
/usr/bin/Command: line 4: sucessfully: command not found
/usr/local/bin/vrecord: eval: line 1182: syntax error near unexpected token `('
/usr/local/bin/vrecord: eval: line 1182: `Error sending command (No error)'
 2020-07-21T12:33:58 - vrecord exited a form in an unexpected way, please report this error to https://github.com/amiaopensource/vrecord/issues
```

**Comment by bturkus** [comment]
slightly different I think?

```
(base) MY-LPAMI-056430:vrecord-deckfix benjaminturkus$ ./vrecord -e
./vrecord: line 1181: Issued: command not found
/usr/bin/Command: line 4: sucessfully: command not found
./vrecord: eval: line 1183: syntax error near unexpected token `('
./vrecord: eval: line 1183: `Error sending command (No error)'
 2020-07-21T14:23:37 - vrecord exited a form in an unexpected way, please report this error to https://github.com/amiaopensource/vrecord/issues
(base) MY-LPAMI-056430:vrecord-deckfix benjaminturkus$ 
```

**Comment by bturkus** [comment]
resolved in #573
